module Result {
  error (input : a) : Result(a, b) {
    `new Err(input)`
  }
}

module A {
  test : String {
    try{x=Result.error("hello")}catch String => a {"blah"}
  }
}
--------------------------------------------------------------------------------
module Result {
  error (input : a) : Result(a, b) {
    `new Err(input)`
  }
}

module A {
  test : String {
    try {
      x =
        Result.error("hello")
    } catch String => a {
      "blah"
    }
  }
}
